cloudinary/__init__.py

```diff
@@ -16,6 +17,15 @@
 ch.setFormatter(formatter)
 logger.addHandler(ch)
 
+SIGNATURE_SHA1 = "sha1"
+SIGNATURE_SHA256 = "sha256"
+SIGNATURE_ALGORITHMS = {
```

Comment:
Maybe move these constants to `utils`, it makes too much noise in the `cloudinary` module.

SIGNATURE_ALGORITHMS looks like a constant, but for dictionaries it's kind of less common. I would make it just a variable or replace with function.